lib/ApplicationFeatures/EnvironmentFeature.cpp

```diff
@@ -81,6 +81,14 @@ void EnvironmentFeature::prepare() {
     uint64_t v = basics::StringUtils::uint64(value);
 
     if (v == 2) {
+#ifdef ARANGODB_HAVE_JEMALLOC
+      LOG_TOPIC("fadc5", WARN, arangodb::Logger::MEMORY)
+          << "/proc/sys/vm/overcommit_memory is set to a value of 2. this setting has been found to be problematic";
+      LOG_TOPIC("d08d6", WARN, Logger::MEMORY)
```

Comment:
Out of curiosity: what is the difference between Logger and arangodb::Logger?

Reply:
Here there is no difference. `Logger` is the class name, and `arangodb::Logger` is the `Logger` class in the `arangodb::namespace`. As the code in the above location is placed inside a `namespace arangodb { ... }` section, it will effectively make no difference, and both variants will be exactly identical.